hr/time-off.md

```diff
@@ -0,0 +1,51 @@
+# Taking time Off
+
+In general, 2i2c inherits its benefits and paid time off policy from {term}`Code for Science and Society`.
+These policies are detailed in the [CS&S Employee Handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing). This page describes some 2i2c-specific practices that are applied in addition to the CS&S policies.
+
+## Goals
+
+- Normalize our PTO policies across countries, so that team members don’t get different PTO depending on their country of residence.
+- Choose PTO policies that minimize the chance that a team member will take *fewer- days off per year than the minimum in their country.
+- Do not apply hard limits and restrictions, but instead encourage soft minimums and coordination amongst team members.
+- Encourage a focus on contributions to the team and organization over number of days worked.
+- Base our numbers off of objective data where possible.
+
+## Policy
+
+- 2i2c does not have a limit on the number of days you may take off each year.
+- We recommend that all 2i2c team members take **at least 40 days** (8 weeks) annually (pro-rated if you started work mid-way through the year).[^time-off]
+- We do not treat holidays differently from vacation days.
```

Comment:
```suggestion
- We do not treat national holidays differently from vacation days.
```

Adding a clarification here as to me "vacation" is the Americanised word for "holiday" and so obviously there's no reason to treat them differently!

Reply:
I now have a better understanding of why you were confused about the difference between `VACATION:` and `HOLIDAY:`, my bad :-)